metabase/metabase
# support parametrized iframe urls

[Related product doc](https://www.notion.so/metabase/Support-variables-in-Iframe-and-Link-cards-15a69354c9018075955ded3076735fcd)

### Description

Supports parametrized iframe urls in Iframe dashboard cards.

### How to verify

- Create a dashboard with an iframe card
- Set the iframe content to `https://example.com/{{var1}}?q={{var2}}`
- Add parameters, connect them to `var1` and `var2`
- Ensure the iframe url uses filter values

### Demo

#### Editing mode
<img width="566" alt="Screenshot 2024-12-17 at 4 13 21 PM" src="https://github.com/user-attachments/assets/e19054f8-624a-4bc9-8128-76b39bf1520d" />

#### Interpolated URL
<img width="755" alt="Screenshot 2024-12-17 at 4 12 58 PM" src="https://github.com/user-attachments/assets/cb54a260-9375-4e9b-b737-3a3aa283ded4" />

### Checklist
